.github/workflows/node-test.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         script:
           - ./scripts/test-hosting.sh
           - ./scripts/test-triggers-end-to-end.sh
-          - npm run test:functional
+          - source ./scripts/set-default-credentials.sh && npm run test:functional
```

Comment:
For sanity could you make a shell script called `./scripts/functional.sh` and then do these two things in there?

Reply:
Cleaned this up in a way.